### PR TITLE
Improve data security and fix for macos

### PR DESCRIPTION
**Pull request by karlinnolabs**
I should probably have added a check to see if 7za is installed (brew install p7zip) might do that later, at the minute I'm busy and can't contribute as much as I'd like.

### CODE REVIEW

**Comment by aconneau** [comment]
This is too verbose and makes it look unnecessary complicated for a single script that's simply downloading some text data.

"If the user is on macOS, download p7zip and use it instead of unzip."
That's the only thing we should handle. 

The checksum is an overkill for this simple script and may bring some additional bugs for other OS.

**Comment by karlinnolabs** [comment]
Given the size of the files, you may think my changes were overly verbose but it was prone to errors - doing a checksum when pulling data from the internet is a fairly standard data security & integrity, also checking the result of unzip before deleting the file is important because in my case when it failed it removed a file which took an hour to download.

**Comment by aconneau** [comment]
You are right. Though this is a research project, downloading well-known datasets from Stanford's website. The security&integrity issue here is not vital. The simpler the script, the better. 
Let's just fix the unzip macOS issue, and put an if statement (if the unzipped .txt file exists) before the "rm" with an error message.

**Comment by karlinnolabs** [comment]
Sure, sounds like a reasonable stop gap between this being a research project and becoming a decanted tool.

File: dataset/get_data.bash [hunk 1]
```diff
@@ -2,30 +2,61 @@
...
+SUMMNLI="d30957eb9e172d0d32aa59cdb197d662"
+
 SNLI='https://nlp.stanford.edu/projects/snli/snli_1.0.zip'
 MultiNLI='https://www.nyu.edu/projects/bowman/multinli/multinli_0.9.zip'
 glovepath='http://nlp.stanford.edu/data/glove.840B.300d.zip'
 
+ZIPTOOL="unzip"
+SUMTOOL="md5sum"
+
+if [ "$OSTYPE" == "OSX" ]; then
```
Comment (aconneau) [inline dataset/get_data.bash]: Did you check that $OSTYPE is actually "OSX" on your machine?
It seems to be "darwin*" for MACs.
https://stackoverflow.com/questions/394230/detect-the-os-from-a-bash-script

**Pull request closed without merging**